src/ActivityRepository.js

```diff
@@ -68,24 +68,13 @@ class ActivityRepository {
     }, 0) / 7).toFixed(0);
   }
 
-  //// ADDITIONAL FEATURE IF WE WANT TO DISPLAY IT
-  calculateDailyCalories(date) {
-    let totalMinutes = this.activityRecord.filter(activity => {
-      return activity.date === date
-    }).reduce((sumMinutes, activity) => {
-      return sumMinutes += activity.minutesActive
-    }, 0);
-    return Math.round(totalMinutes * 7.6);
-  }
-
   compareUserGoalWithCommunityGoal(userGoal, userRepo) {
-    let communityStepGoal = userRepo.getCommunityAvgStepGoal()
+    let communityStepGoal = userRepo.getCommunityAvgStepGoal();
     let goalDifference = userGoal - communityStepGoal;
-    if (goalDifference) { 
-      return `Your goal is ${goalDifference} steps above average!`
+    if (goalDifference > 0) { 
+      return `Your goal is ${goalDifference} steps above average!`;
```

Comment:
Good work combing through the docs for inconsistencies. I think moving forward on other projects, norming on style conventions would be important